source/backend/api/Services/CompensationRequisitionService.cs

```diff
@@ -82,5 +89,44 @@ public bool DeleteCompensation(long compensationId)
             _compensationRequisitionRepository.CommitTransaction();
             return fileFormToDelete;
         }
+
+        private static string GetCompensationRequisitionStatusText(bool? isDraft)
+        {
+            if (isDraft.HasValue)
+            {
+                return isDraft.Value ? "'Draft'" : "'Final'";
+            }
+            else
+            {
+                return "'No Status'";
+            }
+        }
+
+        private void AddNoteIfStatusChanged(long compensationRequisitionId, long acquisitionFileId, (bool? currentStatus, bool? newStatus) statusComparable)
+        {
```

Comment:
I'm okay with using this "tuple" here (statusComparable) but what is the benefit over just having two separate function parameters? ie `AddNoteIfStatusChanged(..., bool? currentStatus, bool? newStatus)`

Reply:
just wrapping a snapshot of these together since either way could be updated before reaching the validation point.